nominal/ts.py

```diff
@@ -394,6 +394,27 @@ def _time_unit_to_conjure(unit: _LiteralTimeUnit) -> api.TimeUnit:
 )
 
 
+def _normalize_timestamp(timestamp: str | datetime | IntegralNanosecondsUTC) -> IntegralNanosecondsUTC:
```

Comment:
you should be able to just use `_SecondsNanos.from_flexible(timestamp).to_nanoseconds()`